python/cuml/dask/extended/linear_model/logistic_regression.py

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common.base import BaseEstimator
+from cuml.common import with_cupy_rmm
+from cuml.common.import_utils import has_daskglm
+
+import cupy as cp
+import numpy as np
+from dask.utils import is_dataframe_like, is_series_like, is_arraylike
+import cudf
+
+
+class LogisticRegression(BaseEstimator):
+    """
+    Distributed Logistic Regression for Binary classification.
+
+
+    Parameters
+    ----------
+    fit_intercept: boolean (default = True)
+       If True, the model tries to correct for the global mean of y.
+       If False, the model expects that you have centered the data.
+    solver : 'admm'
+        Solver to use. Only admm is supported currently.
+    penalty : {'l1', 'l2'} (default = 'l2')
```

Comment:
No `none` or `elasticnet`?

Reply:
Good catch, thanks. The docstring in dask-glm is wrong. Elastic net logistic regression (weighted l1 / l2) is supported. None is not supported, though. The dask-glm implementation requires regularized logistic regression.